build.sh

```diff
@@ -2,7 +2,7 @@
 
 # Define varibles
 SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-DOTNET_VERSION="6.0.100"
+DOTNET_VERSION="6.0.402"
```

Comment:
AppVeyor only has 6.0.400 available

Reply:
Yes adjusted so it installs from global json like Cake does.